code/modules/reagents/chemistry/machinery/pandemic.dm

```diff
@@ -68,117 +79,223 @@
 	if(!wait)
 		. += "waitlight"
 
-/obj/machinery/computer/pandemic/Topic(href, href_list)
+/obj/machinery/computer/pandemic/proc/create_culture(name, bottle_type = "culture", cooldown = 50)
+	var/obj/item/reagent_containers/glass/bottle/B = new/obj/item/reagent_containers/glass/bottle(loc)
+	B.icon_state = "round_bottle"
+	B.pixel_x = rand(-3, 3)
+	B.pixel_y = rand(-3, 3)
+	replicator_cooldown(cooldown)
+	B.name = "[name] [bottle_type] bottle"
+	return B
+
+/obj/machinery/computer/pandemic/ui_act(action, params, datum/tgui/ui, datum/ui_state/state)
 	if(..())
 		return
+	if(inoperable())
+		return
 
-	usr.set_machine(src)
-	if(!beaker) return
+	add_fingerprint(usr)
```

Comment:
```suggestion
```
There is no point in calling this proc again since it is called on attack_hand
![image](https://github.com/ParadiseSS13/Paradise/assets/69762909/8802981e-ebc7-4a26-9cc5-b8914d8213dd)